plugins/optimization-detective/detect.js

```diff
@@ -718,12 +908,16 @@ export default async function detect( {
 		urlMetric.elements.push( elementData );
 		elementsByXPath.set( elementData.xpath, elementData );
 	}
+	breadcrumbedElementsMap.clear();
```

Comment:
Oops. This duplicate call I've removed in https://github.com/WordPress/performance/pull/1968